Descent3/Game2DLL.cpp

```diff
@@ -66,15 +66,15 @@
 
 void SelectNextCameraView(int window);
 #define NUM_CAMERA_VIEWS 3
-extern int Camera_view_mode[NUM_CAMERA_VIEWS];
+extern std::array<int, NUM_CAMERA_VIEWS> Camera_view_mode;
 
 //	Osiris_CreateModuleInitStruct
 //	Purpose:
 //		This function initializes a Module Init Struct with all the needed data to get sent
 //	to the module during initialization.
 extern void Osiris_CreateModuleInitStruct(tOSIRISModuleInit *mi);
 module GameDLLHandle = {NULL};
-extern ddgr_color Player_colors[];
+extern std::array<ddgr_color, 32> Player_colors;
```

Comment:
is there a constant that can be used here instead of `32`?

Reply:
I'm afraid not. It's origin is from [Descent3/renderobject.cpp, line 1479](https://github.com/DescentDevelopers/Descent3/blob/main/Descent3/renderobject.cpp#L1479) which actually doesn't even specify the array a size.

Reply:
I just realized that the size is actually the maximum number of players, so `MAX_NET_PLAYERS` it is.